jerry-debugger/jerry_client_ws.py

```diff
@@ -683,13 +342,162 @@ def __init__(self, address):
     def __del__(self):
         self.client_socket.close()
 
-    def send_message(self, message):
-        size = len(message)
-        while size > 0:
-            bytes_send = self.client_socket.send(message)
-            if bytes_send < size:
-                message = message[bytes_send:]
-            size -= bytes_send
+    def _exec_command(self, command_id):
+        self.send_command(command_id)
+
+    def quit(self):
+        self._exec_command(JERRY_DEBUGGER_CONTINUE)
+
+    def get_continue(self):
+        self._exec_command(JERRY_DEBUGGER_CONTINUE)
+
+    def stop(self):
+        self.send_command(JERRY_DEBUGGER_STOP)
+
+    def finish(self):
+        self._exec_command(JERRY_DEBUGGER_FINISH)
+
+    def next(self):
+        self._exec_command(JERRY_DEBUGGER_NEXT)
+
+    def step(self):
+        self._exec_command(JERRY_DEBUGGER_STEP)
+
+    def memstats(self):
+        self._exec_command(JERRY_DEBUGGER_MEMSTATS)
+
+    def set_break(self, args):
+        _set_breakpoint(self, args, False)
+        if self.breakpoint_info != '':
+            sbp = self.breakpoint_info
+            self.breakpoint_info = ''
+            return sbp
+        return None
```

Comment:
This would be prettier.
```py
_set_breakpoint(self, args, False)

if self.breakpoint_info == '':
  return None

sbp = self.breakpoint_info
self.breakpoint_info = ''
return sbp
```